src/cryptonote_core/blockchain.cpp

```
@@ -299,110 +299,399 @@ uint64_t Blockchain::get_current_blockchain_height(bool lock) const
   if (lock) lock_.lock();
   return m_db->height();
 }
+// --- types used for threaded loader ---
+struct block_data
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: blocks, txs

(cppcoreguidelines-pro-type-member-init)